core/templates/dev/head/pages/exploration_editor/ChangeListService.js

```diff
@@ -0,0 +1,233 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview A standalone service for the exploration editor and collection
```

Comment:
this doesn't explain explicitly about the service.

Reply:
is this really a "standalone" service? 
I think you can just write `A service that maintains a provisional list of changes to be committed to the server.`

Reply:
Thanks. And now?